Specs/Number/Italian/NumberModel.json

```diff
@@ -1,8 +1,178 @@
+      }
+    ]
+  },
+  {
+    "Input": "trenta",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "trenta",
+        "TypeName": "number",
+        "Resolution": {
+          "value": "30"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "trentasei",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "trentasei",
+        "TypeName": "number",
+        "Resolution": {
+          "value": "36"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "trentotto",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "trentotto",
+        "TypeName": "number",
+        "Resolution": {
+          "value": "38"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "quarantatre",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "quarantatre",
+        "TypeName": "number",
+        "Resolution": {
+          "value": "43"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "quarantotto",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "quarantotto",
+        "TypeName": "number",
+        "Resolution": {
+          "value": "48"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "duecento",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "duecento",
+        "TypeName": "number",
+        "Resolution": {
+          "value": "200"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "trecento",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "trecento",
+        "TypeName": "number",
+        "Resolution": {
+          "value": "300"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "ottocento",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "ottocento",
+        "TypeName": "number",
+        "Resolution": {
+          "value": "800"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "duecentoquaranta",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "duecentoquaranta",
+        "TypeName": "number",
+        "Resolution": {
+          "value": "240"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "duecentoquarantatre",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "duecentoquarantatre",
+        "TypeName": "number",
+        "Resolution": {
+          "value": "243"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "eravamo tremila alla festa",
     "NotSupported": "dotnet",
-    "NotSupportedByDesign": "python,javascript,python",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "tremila",
+        "TypeName": "number",
+        "Resolution": {
+          "value": "3000"
+        }
+      }
+    ],
+
```

Comment:
Please be careful with the json specs. This invalid blank line broke the build as Python failed decoding it.